code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfoActivity.java

```diff
@@ -52,6 +52,13 @@ public class AttendeeEventInfoActivity extends AppCompatActivity {
     private void signup(){
         // TODO: update database to add signed-in attendees to event
         // TODO: send to message page: should send to signed-in page instead of checked-in page
+        // sign-in the event and display sign-in message
+        EventManager.checkInToEvent(event.getId(), null);
+        Toast.makeText(this, "Successfully checked in!", Toast.LENGTH_SHORT).show();
```

Comment:
toast message?

Reply:
@Semicarlin I think this is his

Reply:
What I added was passing 'null' to  EventManager.checkInToEvent(event.getId(), null)

Reply:
@ServingCoder can u push that work plz

Reply:
@WhoShotCupidd I am making changes on my computer and will make another commit soon. Should I make my changes in a different manner?

Reply:
ohh nah thats fine just normally if you change something to stop a bug you push it so the reviewer can see that code and see if that actually resolves the issue. Really doesn't matter tho :) @ServingCoder

Reply:
Ok, I'll do two pushes. One with all the comment stuff. Another one that addresses the bug. @WhoShotCupidd